packages/fuels-test-helpers/Cargo.toml

```diff
@@ -10,7 +10,11 @@ description = "Fuel Rust SDK test helpers."
 
 [dependencies]
 fuel-core = { version = "0.8", default-features = false }
+fuel-crypto = "0.5"
 fuel-gql-client = { version = "0.8", default-features = false }
+fuel-tx = "0.10"
```

Comment:
The latest `fuel-tx` is `0.12`. I think this should be updated, otherwise the SDK will be using two different versions.

Reply:
☝️